desktop-exporter/trace_store_test.go

```diff
@@ -89,3 +89,63 @@ func TestAddExceedingTraceLimits(t *testing.T) {
 		assert.Contains(t, store.traceMap, strconv.Itoa(i))
 	}
 }
+
+func TestGetRecentTraceIDs(t *testing.T) {
+	totalTraces := 10
+	numRecentTraceIDs := 5
+	traces := testdata.GenerateOTLPPayload(1, 1, totalTraces)
+
+	store := NewTraceStore(totalTraces)
+	ctx := context.Background()
+	spans := extractSpans(ctx, traces)
+
+	// Assign each span a TraceID derived from its index before adding it to the store
+	// This TraceID is used to validate the ordering of the slice returned by *TraceStore.GetRecentTraceIDs
+	for i, span := range spans {
+		span.TraceID = strconv.Itoa(i)
+		store.Add(ctx, span)
+	}
+
+	recentTraceIDs := store.GetRecentTraceIDs(numRecentTraceIDs)
+
+	// Validate that the number of IDs returned is equal to the lesser value of:
+	// - The number of IDs requested or
+	// - The number traces available in the store
+	if totalTraces < numRecentTraceIDs {
+		assert.Len(t, recentTraceIDs, totalTraces)
+	} else {
+		assert.Len(t, recentTraceIDs, numRecentTraceIDs)
+	}
```

Comment:
Why is this a conditional when these values are fixed constants? I would expect a test for each case